package.json

```diff
@@ -43,16 +42,15 @@
     "lodash-es": "^4.17.11",
     "minimist": "^1.2.0",
     "mocha": "^7.1.2",
+    "semver": "^7.3.2",
     "sinon": "^9.0.2",
     "terser-webpack-plugin": "^3.0.2",
-    "vue": "^2.6.10",
-    "vue-template-compiler": "^2.6.10",
     "webpack": "^4.16.0",
     "webpack-cli": "^3.1.0"
   },
   "engines": {
     "node": ">=8.0.0",
-    "npm": ">=5.7.1"
+    "npm": ">=6.9.0"
```

Comment:
Possibly obsolete now since aliases are no longer in use.

Reply:
Done.